tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
+    - You have relatively new hardware which supports Vulkan.
+    - You are developing a 3D game.
+    - You want to use advanced rendering features, subject to the limitations
+      of mobile hardware.
+
+Choose **Compatibility** if:
+
+    - You are developing for older mobile devices, or older desktop devices. The
+      Compatibility renderer supports the widest range of hardware.
+    - You have older hardware which does not support Vulkan. In this case,
+      Compatibility is the only choice.
+    - You are developing a 2D game, or a 3D game which does not need advanced
+      rendering features.
+
+Keep in mind every game is unique, and this is only a starting point. For example,
+you might choose to use the Compatibility renderer even though you have the latest
+GPU, so you can support the widest range of hardware. Or you might want to use the
+Forward+ renderer for a 2D game, so you can advanced features like compute shaders.
+
+Switching between renderers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the editor, you can always switch between renderers by clicking on the renderer
+name in the upper-right corner of the editor.
+
+Switching between renderers may require some manual tweaks to your scene, lighting,
+and environment, since each renderer is different. In general, switching between
+the Forward+ or Mobile renderers and the Compatibility renderer will require more
+adjustments than switching between the Mobile and Forward+ renderers.
+
+Switching between different renderers at runtime is not yet supported.
+
+Feature comparison
+------------------
+
+This is not a complete list of the features of each renderer. If a feature is
+not listed here, it is available in all renderers, though it may be much faster
+on some renderers. For a list of *all* features in Godot, see :ref:`doc_list_of_features`. 
+
+Overall comparison
+~~~~~~~~~~~~~~~~~~
+
++---------------------+--------------------------+--------------------------+--------------------------+
+| Feature             | Compatibility            | Mobile                   | Forward+                 |
++=====================+==========================+==========================+==========================+
+| **Required**        | Older or low-end.        | Newer or high-end.       | Newer or high-end.       |
+| **hardware**        |                          | Requires Vulkan support. | Requires Vulkan support. |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Runs on new hardware| ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Runs on old and     | ✔️ Yes.                  | ❌ No.                   | ❌ No.                   |
+| low-end hardware    |                          |                          |                          |
```

Comment:
Forward+/Mobile run on low-end hardware, but they won't run as well as Compatibility.

Mobile runs better on low-end hardware compared to Forward+.

Reply:
Sure. In that case, I think there should be another line in the table specifically for requiring Vulkan/D3D12/Metal support. The editor not running at all is a hard limit that deserves a solid X.